packages/data/src/redux-store/combine-reducers.js

```diff
@@ -0,0 +1,17 @@
+export function combineReducers( reducers ) {
```

Comment:
This is more or less a simplified version of the Redux `combineReducers()`: https://github.com/reduxjs/redux/blob/957aed6e23624113b4d0ff6d9f277f25fca90b38/src/combineReducers.ts#L123

Should we add a mention in CREDITS.md?

Reply:
Probably not, I think it's sufficiently different from Redux. There are only so many ways to write a `combineReducers` function, it's very hard to be completely original 🙂 Also, Gutenberg doesn't even have a `CREDITS.md` file.

Reply:
Fair enough 👍